modules/serverless-rn-1-20-0.adoc

```diff
@@ -0,0 +1,146 @@
+
+[id="known-issues-1-20-0_{context}"]
+== Known issues
+
+* {ServerlessProductName} deploys Knative services with a default address that uses HTTPS. When sending an event to a resource inside the cluster, the sender does not have the cluster certificate authority (CA) configured. This causes event delivery to fail, unless the cluster uses globally accepted certificates.
++
+For example, an event delivery to a publicly accessible address works:
++
+[source,terminal]
+----
+$ kn event send --to-url https://ce-api.foo.example.com/
+----
++
+On the other hand, this delivery fails if the service uses a public address with an HTTPS certificate issued by a custom CA:
++
+[source,terminal]
+----
+$ kn event send --to Service:serving.knative.dev/v1:event-display
+----
++
+Sending an event to other addressable objects, such as brokers or channels, is not affected by this issue and works as expected.
+
+* The Kafka broker currently does not work on a cluster with Federal Information Processing Standards (FIPS) mode enabled.
+
+* If you create a Springboot function project directory with the `kn func create` command, subsequent running of the `kn func build` command fails with this error message:
++
+[source,terminal]
+----
+[analyzer] no stack metadata found at path ''
+[analyzer] ERROR: failed to : set API for buildpack 'paketo-buildpacks/ca-certificates@3.0.2': buildpack API version '0.7' is incompatible with the lifecycle
+----
++
+As a workaround, you can change the `builder` property to `gcr.io/paketo-buildpacks/builder:base` in the function configuration file `func.yaml`.
+
+* Deploying a function using the `gcr.io` registry fails with this error message:
++
+[source,terminal]
+----
+Error: failed to get credentials: failed to verify credentials: status code: 404
+----
++
+As a workaround, use a different registry than `gcr.io`, such as `quay.io` or `docker.io`.
+
+* Functions created with the TypeScript runtime and the `http` template do not get deployed on the cluster.
++
+This issue can be reproduced with these steps:
++
+[source,terminal]
+----
+$ kn func create -l typescript -t http /tmp/ts-http
+Created typescript Function in /tmp/ts-http
+----
++
+[source,terminal]
+----
+$ kn func deploy -r docker.io/jrangelramos -p /tmp/ts-http
+   🙌 Function image built: docker.io/jrangelramos/ts-http:latest
+🕖 Deploying function to the cluster
+Error: knative deployer failed to wait for the Knative Service to become ready: timeout: service 'ts-http' not ready after 120 seconds
+----
++
+[source,terminal]
+----
+$ kn func run -p /tmp/ts-http
+cat: /sys/fs/cgroup/memory/memory.limit_in_bytes: No such file or directory
+/layers/paketo-buildpacks_node-engine/node/profile.d/0_memory_available.sh: line 3: / ( 1024 * 1024 ) : syntax error: operand expected (error token is "/ ( 1024 * 1024 ) ")
+Error: Cannot find module '/workspace/build/index.js'
+Require stack:
+
+    /layers/paketo-buildpacks_npm-install/modules/node_modules/faas-js-runtime/bin/cli.js
+----
```

Comment:
Here I think there is no need to add the steps to reproduce because the issue description is self descriptive. 

On reproduce steps it has the `kn func run` (just as I added in the ticket), but I added that way just because it is the quickest way to capture the error (devs could use that for throubleshoot). End users would do `kn func deploy` and they would need to inspect the console for the error on pod logs (which is created and destroyed pretty quick).